build/migration/bin/2_previous-integration-test.sh

```diff
@@ -1,19 +1,19 @@
 #!/usr/bin/env bash
 
 ###############################################################################
-# (C) Copyright IBM Corp. 2021
+# (C) Copyright IBM Corp. 2021, 2022
 #
 # SPDX-License-Identifier: Apache-2.0
 ###############################################################################
 
 set -ex
 
 run_tests(){
-    # The integration tests may be overriden completely, or fall through to the default. 
+    # The integration tests may be overriden completely, or fall through to the default.
     migration="${1}"
 
     if [ ! -z "${migration}" ] && [ -f "build/migration/${migration}/2_previous-integration-test.sh" ]
```

Comment:
I know it's not part of this PR, but we should try and use `[[ ... ]]` for tests because they are then evaluated within the shell.